controllers/org.js

```diff
@@ -72,15 +73,15 @@ exports.createOrg = function(req, res) {
       website: req.body.orgWebsite,
       facebook: req.body.orgFacebook,
       twitter: req.body.orgTwitter,
-//      survey: req.body.orgSurvey, //TODO this should be added later
       subdomain: req.body.orgSubdomain
     });
     newOrg.save(function(err) {
       if(!err) {
         User.findById(req.user.id, function(err, theUser) {
           theUser.org = newOrg._id;
```

Comment:
Interesting! This line doesn't actually work because we did not define a second argument to the callback function on line 78